test/extended/util/annotate/rules.go

```diff
@@ -13,180 +20,40 @@ var (
-		},
 		// tests that are known broken and need to be fixed upstream or in openshift
 		// always add an issue here
 		"[Disabled:Broken]": {
-			`mount an API token into pods`,                                // We add 6 secrets, not 1
-			`ServiceAccounts should ensure a single API token exists`,     // We create lots of secrets
-			`unchanging, static URL paths for kubernetes api services`,    // the test needs to exclude URLs that are not part of conformance (/logs)
-			`Services should be able to up and down services`,             // we don't have wget installed on nodes
-			`Network should set TCP CLOSE_WAIT timeout`,                   // possibly some difference between ubuntu and fedora
-			`\[NodeFeature:Sysctls\]`,                                     // needs SCC support
-			`should check kube-proxy urls`,                                // previously this test was skipped b/c we reported -1 as the number of nodes, now we report proper number and test fails
-			`SSH`,                                                         // TRIAGE
-			`should implement service.kubernetes.io/service-proxy-name`,   // this is an optional test that requires SSH. sig-network
 			`should idle the service and DeploymentConfig properly`,       // idling with a single service and DeploymentConfig
 			`should answer endpoint and wildcard queries for the cluster`, // currently not supported by dns operator https://github.com/openshift/cluster-dns-operator/issues/43
-			`should allow ingress access on one named port`,               // https://bugzilla.redhat.com/show_bug.cgi?id=1711602
-			`recreate nodes and ensure they function upon restart`,        // https://bugzilla.redhat.com/show_bug.cgi?id=1756428
-			`\[Driver: iscsi\]`,                                           // https://bugzilla.redhat.com/show_bug.cgi?id=1711627
-
-			"RuntimeClass should reject",
-
-			`Services should implement service.kubernetes.io/headless`,       // requires SSH access to function, needs to be refactored
-			`ClusterDns \[Feature:Example\] should create pod that uses dns`, // doesn't use bindata, not part of kube test binary
-			`Simple pod should handle in-cluster config`,                     // kubectl cp doesn't work or is not preserving executable bit, we have this test already
-
-			// TODO(node): configure the cri handler for the runtime class to make this work
-			"should run a Pod requesting a RuntimeClass with a configured handler",
-			"should reject a Pod requesting a RuntimeClass with conflicting node selector",
-			"should run a Pod requesting a RuntimeClass with scheduling",
-
-			// A fix is in progress: https://github.com/openshift/origin/pull/24709
-			`Multi-AZ Clusters should spread the pods of a replication controller across zones`,
-
-			// Disabled as per networking team. Follow-up tracked via https://bugzilla.redhat.com/show_bug.cgi?id=1861214
-			`EndpointSliceMirroring should mirror a custom Endpoints resource through create update and delete`,
```

Comment:
you went through all of these and made sure that only match kube tests?

Reply:
The reverse. The rules in this file only match origin tests. As per the comment at the top of the file, the openshift/kubernetes equivalent only matches kube tests.